src/qt/overviewpage.cpp

```diff
@@ -178,6 +176,7 @@ void OverviewPage::handleTransactionClicked(const QModelIndex &index)
 void OverviewPage::setPrivacy(bool privacy)
 {
     m_privacy = privacy;
+    const auto& m_balances = walletModel->getCachedBalance();
```

Comment:
061c6ee98af7185fa340989da8178e0149207e3f

`m_` prefix is used for member variables, not for local ones.

See https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-c